NOTES-RELEASE.md

```diff
@@ -0,0 +1,13 @@
+
+###v0.1.0
+ - Determine programming tools and requirements.
+ - Create graphical base user interface.
+ - Implement sheetstorm user authentication.
+ - Entering the possibility of changing the password for users.
+ - Users are allow to join project.
+ - Show accurate hour display
+ - Description field is ready to prepare reports.
+ - Admin is allow to manage employees list.
+ - Manager is allow to see projects and manage them.
+ - Markdown support to description field.
+ - Users are possible to set and change account details.
```

Comment:
Also please add to all this notes numbers of pull requests where it is from.

Reply:
How should it look like?

Something like that?
> - Markdown support to description field. #37

Reply:
> How should it look like?
> 
> Something like that?
> 
> > * Markdown support to description field. #37

Almost yes :) I would like to have it written with brackets, like this:

``` * Markdown support to description field. (#37) ```

Reply:
I will try to do it tomorrow ;)